src/components/GrowthChart/GrowthChart.tsx

```diff
@@ -84,18 +84,21 @@ export const GrowthChart = ({
                     />
                 </div>
             </div>
-
-            <GrowthChartBuilder
-                measurementData={measurementData}
-                datasetValues={dataSetValues}
-                datasetMetadata={dataSetMetadata}
-                yAxisValues={yAxisValues}
-                keysDataSet={keysDataSet}
-                dateOfBirth={new Date(trackedEntity?.dateOfBirth)}
-                category={category}
-                dataset={dataset}
-                isPercentiles={isPercentiles}
-            />
+            <div className='px-2'>
```

Comment:
You can remove one of these divs, as done in the suggestion:

```suggestion
 <div className='px-2 overflow-auto'>
                <GrowthChartBuilder
                    measurementData={measurementData}
                    datasetValues={dataSetValues}
                    datasetMetadata={dataSetMetadata}
                    yAxisValues={yAxisValues}
                    keysDataSet={keysDataSet}
                    dateOfBirth={new Date(trackedEntity?.dateOfBirth)}
                    category={category}
                    dataset={dataset}
                    isPercentiles={isPercentiles}
                />
 </div>